### PR TITLE
Feature/map widget functionality

### PR DESCRIPTION
**Pull request by ysdevys**
added map widget functionality:
set multiple markers

### CODE REVIEW

**Comment by syncall** [comment]
But the feature is not done yet right?

File: index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
```
Comment (syncall) [inline index.html]: Can you import the css with webpack at your map Class (import ...)?
There is probably an npm module you can download.

**Review by syncall — DISMISSED** [review]
But the feature is not done yet, rigth?

**Comment by ysdevys** [comment]
css problem fixed, custom marker added
image of marker can be discussed

**Comment by chaoran-chen** [comment]
Hey guys. [Here is an online demo of this widget.](http://54.68.164.199:8080/map/)

Currently, there are some [sample markers](https://github.com/MusicConnectionMachine/VisualizationG6/pull/66/files#diff-8702fd2db8ba3ac56fedef8848718a80R49) but what actually should be displayed? There seems to be no request to the data and relationship groups.
(also @kordianbruck)

**Comment by kordianbruck** [comment]
@chaoran-chen I don't think much other than simple markers was implemented yet. So basically we need to hook up the data from the API (Create an endpoint for that possibly?) and then display them in the map. 
(Feedback from other tutors should be coming tomorrowish to specify this more)

For now please merge this into your repo and allow the map to be equally used as the other widgets. Maybe @ysdevys can open up a pull request there?

**Comment by chaoran-chen** [comment]
Do we have even geographical data? How do they look like? I thought that we already have feature freeze..

**Comment by pfent** [comment]
@chaoran-chen We don't necessarely have something like geo coordinates, but something like a "place of birth" and a "place of death".

If I remember the sample data correctly, the data isn't necessarely precise, nor consistent. I.e. "Italy" as well as "Salzburg" are places of birth

**Comment by kordianbruck** [comment]
@sacdallago can answer how to reverse lookup GeoData

**Comment by chaoran-chen** [comment]
Just to clarify: that means data about locations are already / will be extracted and only an API-endpoint is missing? Are there any sample data? I know those from https://github.com/MusicConnectionMachine/RelationshipsG3/issues/21 and https://github.com/MusicConnectionMachine/RelationshipsG4/issues/49 but they don't contain locations.

How will they be stored in the database? (Who knows it, whom can I ask?)

**Comment by sacdallago** [comment]
@chaoran-chen I would say @TimHenkelmann , @LukasNavickas or @ShilpaGhanashyamGore will have an idea about this, as well as @lgfrbcsgo and @simonzachau if these things pop up anywhere else in the data (e.g.: common crawl).

and [look here](https://github.com/PokemonGoers/PokeData/issues/68#issuecomment-244857683) :)

**Comment by ysdevys** [comment]
Hi guys
the map widget is just a widget and displays map with custom marker icons and the list of markers lat/long coordinates to be shown as a property when you call the Map widget.
The task was to do a widget.
If you know how and what I should display, please tell and I can implement it.
Otherwise I think the map widget as such is done? Right?

**Comment by pfent** [comment]
@ysdevys The widget itself looks fine, we're just trying to connect the widget to the API, so it is actually displaying something useful 😉

**Comment by ysdevys** [comment]
It requires Latlon data for now but i can check if it can work directly with city names.
I just have exams this week...

**Comment by goldbergtatyana** [comment]
The widget looks very nice! Very curious seing it being hooked up with real data

**Comment by kordianbruck** [comment]
Merged here: https://github.com/MusicConnectionMachine/VisualizationG5/pull/68

**Pull request closed without merging**